nilearn/_utils/testing.py

```diff
@@ -186,14 +186,15 @@ def __call__(self, *args, **kwargs):
             basename = os.path.basename(fname)
             if basename in self.csv_files:
                 array = self.csv_files[basename]
+
                 # np.savetxt does not have a header argument for numpy 1.6
```

Comment:
Another one to keep in mind when we move the numpy requirement to 1.6. If you have some spare cycles, feel free to open an issue as you mentioned so that we don't forget about these things and reference #397 inside.